code/src/java/pcgen/cdom/processor/ChangeArmorType.java

```diff
@@ -148,10 +136,6 @@ public boolean equals(Object obj)
 			return false;
 		}
 		ChangeArmorType other = (ChangeArmorType) obj;
-		if (result == null)
```

Comment:
Agree with this change based on the constructor, but it indicates a lot of the docs in the file are now old (it says a lot about null being legal and things being removable), so there are a lot of other changes that can occur in this file, and the docs should be updated as well.